src/mpid/ch4/netmod/ofi/ofi_rma.h

```diff
@@ -876,7 +876,7 @@ static inline int MPIDI_OFI_do_accumulate(const void *origin_addr,
                                           MPIDI_av_entry_t * addr, MPIR_Request ** sigreq)
 {
     int rc, mpi_errno = MPI_SUCCESS;
-    uint64_t flags;
+    uint64_t flags = 0;
     MPIDI_OFI_win_request_t *req = NULL;
```

Comment:
I have been hesitating on this type of warning-fix. The compilers warns us when it thinks there is a code path lead to variable being used without set (I assume that's the warning). By blindly setting initial value, it is equivalent to disable that particular warning. I.E. in the case that we do have code logic error, the compiler will not warn us anymore. We are just replacing the potential use-uninitialized error with potential use-wrong-value error.

The proper fix, IMHO, is to address why the compiler failed to figure out the code path. Sometime this is difficult due to our legacy frame code or compiler being too old or too silly. When that is too difficult, I would suggest at least move the (often) temporary variable close to where it is being used and maybe also add a comment explaining the code path. That way, it is a less blind fix.

Reply:
That's a fair point and I agree.  Let me spend some more time on it.

Reply:
I'm not able to reproduce this warning at all.  Very strange.  I'll drop this part of the patch.